Automattic/woocommerce-payments
# Update account connection success message

Fixes #4747

#### Changes proposed in this Pull Request
- Update connection success notice to match the new design. Using a different copy after PO and full KYC.
- Move it down, so it's rendered close to the other cards, leaving the rest of possible notices at the top.

With PO thresholds notices being handled in the two other issues, and the new "Account" card design being out of the MVP, I think this the only remaining UI change to be done. Do you agree?

#### Testing instructions
- Go to **Payments → Overview** and add `&wcpay-connection-success=1` to the URL.
- The `After Full KYC` notice should be visible.
- Edit [get_account_status_data](https://github.com/Automattic/woocommerce-payments/blob/f61304b494a35eb7d394030fb3a43fc8d3c0118d/includes/class-wc-payments-account.php#L228) to return `isEnabled => true` and `isComplete => false`, and refresh.
- The `After PO` notice should be visible.

You can also test it by onboarding a new account, but this PR only updates the UI, so no for the logic to be tested.

#### Screenshots
<table>
<tr><th>Before</th><td><img width="698" alt="Screenshot 2023-03-10 at 15 47 53" src="https://user-images.githubusercontent.com/7670276/224346145-82215d8d-edde-4113-bd51-02f672a1c348.png"></td>
<tr><th>After PO</th><td><img width="691" alt="po" src="https://user-images.githubusercontent.com/7670276/225875927-5ddb7917-f201-4299-9b5d-37bca7644c26.png"></td>
<tr><th>After Full KYC</th><td><img width="691" alt="full" src="https://user-images.githubusercontent.com/7670276/225875912-3a1a6b54-ea9c-4827-b7b2-ad8a67a3f7b1.png"></td>
</table>



-------------------


**Post merge**
